### ISSUE TITLE
CalendarActivity, AddCalendarEntryActivity

### ISSUE THREAD

**Issue by matalie** [Moeby/FlatsharePlanner#58]
- show calendar xml
- button: redirect to add_calendar_entry xml and show this

**UserStories**
__#24 Access Calendar US9__

(#25 Calendar shows assignment  US10
#26 Create Calendar Entry US11
#34 Categorize Calendar Entry US19
#35 Assign Calendar Entry US20)
  
To do:
doublecklick on date --> open AddCalendarEntryActivity with correct date

**Comment by Moeby** [comment]
This has not been implemented yet so I'm moving this back into "in progress". 

@ThothHermes : Try to find a different calendar library which can show events (as we discussed )and use that in the application. Needs to be done until Wednesday

**Comment by teshi24** [comment]
Discussion with @matalie @ThothHermes:
Calendar library will be changed in the next sprint in another ticket.
We still need to figure out how exactly to implement the adding when the view of the CalendarItems in the Calendar is correctly implemented.

Needed implementation:
- get intent extras in addCalendarItemActivity
